src/clzones.cpp

```diff
@@ -18,53 +18,110 @@
 zone_manager::zone_manager()
 {
     types.emplace( zone_type_id( "NO_AUTO_PICKUP" ),
-                   zone_type( translate_marker( "No Auto Pickup" ) ) );
-    types.emplace( zone_type_id( "NO_NPC_PICKUP" ), zone_type( translate_marker( "No NPC Pickup" ) ) );
-    types.emplace( zone_type_id( "LOOT_UNSORTED" ), zone_type( translate_marker( "Loot: Unsorted" ) ) );
-    types.emplace( zone_type_id( "LOOT_FOOD" ), zone_type( translate_marker( "Loot: Food" ) ) );
-    types.emplace( zone_type_id( "LOOT_PFOOD" ), zone_type( translate_marker_context( "perishable food",
-                   "Loot: P.Food" ) ) );
-    types.emplace( zone_type_id( "LOOT_DRINK" ), zone_type( translate_marker( "Loot: Drink" ) ) );
+                   zone_type( translate_marker( "No Auto Pickup" ),
+                              translate_marker( "PC doesn't Auto Pickup items inside the zone" ) ) );
+    types.emplace( zone_type_id( "NO_NPC_PICKUP" ),
+                   zone_type( translate_marker( "No NPC Pickup" ),
+                              translate_marker( "Friendly NPCs don't pickup items inside the zone" ) ) );
+    types.emplace( zone_type_id( "LOOT_UNSORTED" ),
+                   zone_type( translate_marker( "Loot: Unsorted" ),
+                              translate_marker( "Source of items for Sort out loot Zone action. It can overlap with Loot zones of different types." ) ) );
+    types.emplace( zone_type_id( "LOOT_FOOD" ),
+                   zone_type( translate_marker( "Loot: Food" ),
+                              translate_marker( "Destination for comestibeles. If more specific food zone is not defined, all food is moved here." ) ) );
+    types.emplace( zone_type_id( "LOOT_PFOOD" ),
+                   zone_type( translate_marker_context( "perishable food", "Loot: P.Food" ),
+                              translate_marker( "Destination for perishable comestibles. Does include perishable drinks if such zone is not specified." ) ) );
+    types.emplace( zone_type_id( "LOOT_DRINK" ),
+                   zone_type( translate_marker( "Loot: Drink" ),
+                              translate_marker( "Destination for drinks. Does include perishable drinks if such zone is not specified." ) ) );
     types.emplace( zone_type_id( "LOOT_PDRINK" ),
-                   zone_type( translate_marker_context( "perishable drink", "Loot: P.Drink" ) ) );
-    types.emplace( zone_type_id( "LOOT_GUNS" ), zone_type( translate_marker( "Loot: Guns" ) ) );
+                   zone_type( translate_marker_context( "perishable drink", "Loot: P.Drink" ),
+                              translate_marker( "Destination for perishable drinks." ) ) );
+    types.emplace( zone_type_id( "LOOT_GUNS" ),
+                   zone_type( translate_marker( "Loot: Guns" ),
+                              translate_marker( "Destination for guns." ) ) );
```

Comment:
`guns and archery weapons`? I don't know how this zone works, but slingshots, bows, and crossbows are all categorized as guns in-game, so if the zone also does it that way it would help to clarify it.